docs/rules/component-name-unique.md

```diff
@@ -0,0 +1,109 @@
+# component-name-unique
+
+Verifies component names are unique.
+
+|OAS|Compatibility|
+|---|---|
+|2.0|✅|
```

Comment:
Is it compatible with 2.0? It looks from the file organization that is is for 3.0, and 3.1?

Reply:
Removed 2.0 compatibility now in the documentation 👍